Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ruby:2.5.8-alpine
+FROM ruby:2.6.0-alpine
```

Comment:
2.6.10-alpine makes some errors?

Reply:
There is no error. So updated to 2.6.10 !